include/opentracing/span.h

```diff
@@ -27,10 +28,27 @@ class SpanContext {
       const = 0;
 };
 
+struct LogRecord {
+  using Field = std::pair<std::string, Value>;
+
+  SystemTime timestamp;
+  std::vector<Field> fields;
+};
+
 // FinishOptions allows Span.Finish callers to override the finish
 // timestamp.
 struct FinishSpanOptions {
   SteadyTime finish_steady_timestamp;
+
+  // log_records allows the caller to specify the contents of many Log() calls
+  // with a single vector. May be empty.
+  //
+  // None of the LogRecord.timestamp values may be SystemTime() (i.e., they must
+  // be set explicitly). Also, they must be >= the Span's start system timestamp
+  // and <= the finish_steady_timestamp converted to system timestamp
+  // (or SystemTime::now() if finish_steady_timestamp is default-constructed).
+  // Otherwise the behavior of FinishWithOptions() is undefined.
```

Comment:
> ```
> // Otherwise the behavior of FinishWithOptions() is undefined.
> ```
Can we change this to unspecified or remove that line. [Undefined behavior](https://www.quora.com/What-is-the-difference-between-undefined-unspecified-and-implementation-defined-behavior) would mean a tracer is free to do anything (abort, segfault, etc). Given that converting between steady and system time is a noisy process, I  don't think it would be right for a tracer to abort or corrupt the process if a timestamp is out of bounds.